langchain/agents/conversational_chat/output_parser.py

```diff
@@ -19,12 +19,14 @@ def parse(self, text: str) -> Union[AgentAction, AgentFinish]:
         if "```" in cleaned_output:
             cleaned_output, _ = cleaned_output.split("```")
         if cleaned_output.startswith("```json"):
-            cleaned_output = cleaned_output[len("```json") :]
+            cleaned_output = cleaned_output[len("```json"):]
         if cleaned_output.startswith("```"):
-            cleaned_output = cleaned_output[len("```") :]
+            cleaned_output = cleaned_output[len("```"):]
         if cleaned_output.endswith("```"):
             cleaned_output = cleaned_output[: -len("```")]
-        cleaned_output = cleaned_output.strip()
+
+        # sometimes, two backquotes in text.
+        cleaned_output = cleaned_output.rstrip("`").strip()
```

Comment:
rstrip only supported in python >=3.9 i believe and we still officially support 3.8